tests/unit/dummy.php

```diff
@@ -1,7 +1,7 @@
 <?php
-
-// TODO: Remove this file once there are actual tests.
-
+/**
```

Comment:
Should this be this?

```suggestion
/*
```

Reply:
The coding standards don't allow it somehow. It's viewed as the class comment so must start with `/**`. This will be removed soon once we add tests.